plugins/tools/config/policy.json

```diff
@@ -6,5 +6,6 @@
   "tools:universal_search_role_assignments": "rule:cloud_admin or role:cloud_support_tools_viewer",
   "tools:universal_search_user_role_assignments": "rule:cloud_admin or role:cloud_support_tools_viewer or user_id:%(user.id)s",
   "tools:switch_to_project": "not project_id:%(project.id)s",
-  "tools:show_scope_object": "rule:cloud_admin or role:cloud_support_tools_viewer"
+  "tools:show_scope_object": "rule:cloud_admin or role:cloud_support_tools_viewer",
+  "metrics:application_gaas": "role:cloud_support_tools_viewer"
```

Comment:
Please add this line into the policy file of the metrics plugin, not here. The metrics policy file is here: https://github.com/sapcc/elektra/blob/master/plugins/metrics/config/policy.json